trendico/models.py

```diff
@@ -1,3 +1,48 @@
 from django.db import models
-
+from datetime import datetime, timedelta
 # Create your models here.
+CATEGORY_CHOICES = (
+    ('L', 'Laptop'),
+    ('T', 'Tab'),
+    ('C', 'Camera'),
+    ('H', 'Headphone')
+)
+class Product(models.Model):
+    product_name=models.CharField(max_length=50)
+    img_URL = models.ImageField(upload_to='images')
+    price = models.FloatField()
+    discount_price = models.FloatField(blank=True, null=True)
+    category = models.CharField(choices=CATEGORY_CHOICES, max_length=1)
+    description = models.TextField()
+    total_quantity=models.PositiveIntegerField()
+    created_at=models.DateTimeField(auto_now_add=True)
```

Comment:
the better way of created_at updated_at would be:
create a model:

`class BaseModel(models.Model):
    """Provide default fields that are expectedly to be needed by almost all models"""

    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)
    is_active = models.BooleanField(default=True)
    history = HistoricalRecords(inherit=True)

    class Meta:
        abstract = True`
        
      then use this for all of your models e.g `class Product(BaseModel):`
      this will add timestamps and ability to hide any product at anytime using is_active field